apps/blockchain/starknet/src/bridge.cairo

```diff
@@ -367,6 +370,10 @@ mod bridge {
             self.l2_to_l1_addresses.write(collection_l2, collection_l1);
             self.emit(L1L2CollectionMappingUpdated { collection_l1, collection_l2 });
         }
+
+        fn is_token_escrowed(ref self: ContractState, collection: ContractAddress, token_id: u256) -> bool {
+            !self.escrow.read((collection, token_id)).is_zero()
+        }      
```

Comment:
By the way I'm not sure this view is really needed since it's only use for testing purpose.
Please see https://foundry-rs.github.io/starknet-foundry/testing/testing-contract-internals.html for a better solution.